netbox/dcim/models/devices.py

```diff
@@ -1098,7 +1098,10 @@ def vc_interfaces(self, if_master=True):
 
         :param if_master: If True, return VC member interfaces only if this Device is the VC master.
         """
-        filter = Q(device=self)
+        if self.pk:
+            filter = Q(device=self)
+        else:
+            filter = Q()
```

Comment:
What prompted this change? Is filtering by None no longer supported? If that's the case I suspect there are many other cases that may need to be updated.

Reply:
this issue https://forum.djangoproject.com/t/unsaved-model-instances-to-related-filters-in-tests-for-django-5-0/26586/2 from "Passing unsaved model instances to related filters is no longer allowed."